conjur/api.py

```diff
@@ -285,6 +284,46 @@ def update_policy_file(self, policy_id, policy_file):
 
         return self._load_policy_file(policy_id, policy_file, HttpVerb.PATCH)
 
+    def rotate_anothers_api_key(self, resource, resource_to_rotate):
+        """
+        This method is used to rotate a user/host's API key
+        """
+        # Attach the resource type (user or host)
+        full_resource_id=':'.join([resource, resource_to_rotate])
+        query_params = {
+            'role': full_resource_id
+        }
+        response = invoke_endpoint(HttpVerb.PUT, ConjurEndpoint.ROTATE_API_KEY,
+                                   self._default_params,
+                                   api_token=self.api_token,
+                                   ssl_verify=self._ssl_verify,
+                                   query=query_params).text
+        return response
+
+    def rotate_personal_api_key(self, resource_to_rotate, current_password):
+        """
+        This method is used to rotate a personal API key
+        """
+        response = invoke_endpoint(HttpVerb.PUT, ConjurEndpoint.ROTATE_API_KEY,
+                                   self._default_params,
+                                   api_token=self.api_token,
+                                   auth=(resource_to_rotate, current_password),
+                                   ssl_verify=self._ssl_verify).text
+        return response
+
+    def change_password(self, resource_password_to_change, current_password, new_password):
```

Comment:
isn't `resource_password_to_change` always equal user?
Why pass it if you know it?

Reply:
same applies here. https://github.com/cyberark/conjur-api-python3/pull/148#discussion_r563290330

it should be the name of the logged in user not `user`